GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FilePreviewView.swift

```diff
@@ -0,0 +1,232 @@
+#if canImport(QuickLookThumbnailing)
+    import QuickLookThumbnailing
+#endif
+
+import UIKit
+
+extension SecureConversations {
+    class FilePreviewView: UIView {
```

Comment:
Can be final?

Reply:
Added here 26bd52271dfcf47c510b4bbae624baa646c8fe43.